projects/schematics/src/shared/utils/new-module-utils.ts

```diff
@@ -212,3 +212,25 @@ function elementExists(
 
   return false;
 }
+
+export function getModule(sourceFile: SourceFile): CallExpression | undefined {
```

Comment:
Can we refactor the `addToModuleInternal()` from this file to use this new helper function?